pages/dashboard/dashboard-page.js

```diff
@@ -680,6 +681,7 @@ exports.DashboardPage = class DashboardPage extends BasePage {
       (await this.pluginModalHeader.isVisible()) &&
       (await this.pluginModalContinueBtn.isVisible())
     ) {
+      await this.clickOnPluginContinueBtn();
```

Comment:
This would work for this release, but would lead to a flaky test whenever there weren't two screens to "Continue" forcing us to update it recurrently.

I'm looking for another approach to this problem.

Reply:
This is a temporary solution to prevent the tests from failing for now

Reply:
Yes, but we can make it work for any future similar case, regarding new features pop ups.
How do you see [these changes](https://github.com/penpot/penpotqa/commit/3be0255117c22de8821bcc0214afd6ebc1d744bd)?